### DISCUSSION TITLE
Add Annotated Wireframes for Semantics

### DISCUSSION THREAD

**Discussion by mgifford** [uswds/uswds#4931] [Ideas]
For designs to be effectively conveyed to developers, accessibility annotation is critical. Making sure both parties are clear what functionality a given component is supposed to have depending on mouse/keyboard interactions for instance. 

Some ideas are spelled out here that I think are worth implementing for the USWDS. 

Having common Figma/Sketch/Penpot files with accessibility annotation would be great.  

https://medium.com/designing-atlassian/create-accessible-designs-using-the-figma-a11y-annotation-kit-35371f00dac5

**Comment by amyleadem** [comment]
Thanks for sharing this, @mgifford. It's always good to get a11y in the discussion as early as possible. 

While USWDS does not have an official Figma implementation, there are still intriguing ideas here about new ways to integrate accessibility considerations into the design and development process. Accessibility annotation could be a great consideration for future design work.

**Reply by nloand** [reply]
@amyleadem will there be a Figma design kit since Adobe is in the process of acquiring Figma? Adobe XD is no longer an option in the stand alone catalog from Adobe.

**Reply by amyleadem** [reply]
Hi @nloand, thanks for reaching out. This is something we are looking into, but we are waiting for Figma FedRAMP authorization before moving forward. Please let me know if you have any other questions!
